django_app/settings.py

```diff
@@ -163,6 +163,17 @@
     },
 ]
 
+# DRF global settings
+REST_FRAMEWORK = {
+    "DEFAULT_AUTHENTICATION_CLASSES": [
+        "rest_framework.authentication.TokenAuthentication",
+    ],
+    "DEFAULT_PERMISSION_CLASSES": [
+        "rest_framework.permissions.IsAuthenticated",
+        "rest_framework.permissions.IsAdminUser",
+    ],
+    "DATETIME_FORMAT": "%Y-%m-%d %H:%M:%S",
+}
 
```

Comment:
Most of the expected users are from the united states, hence the date/time format should sync so as not to have any complexity on the app, especially on something as basic as date/time.
Kindly consider using this format: “mm-dd-yyyy”

Reply:
@DeFidelity Nah. The expected users are international, since I am crowdsourcing annotators globally.

YYYY-MM-DD is NEVER confused. Also, it sorts easily.

Reply:
True,